tamr_toolbox/data_io/df_connect/client.py

```diff
@@ -317,6 +326,44 @@ def export_dataset(
     return r.json()
 
 
+def export_delta_dataset(
+    connect_info: Client,
+    *,
+    dataset_name: str,
+    target_table_name: str,
+    from_version: str = None,
+    to_version: str = None,
+    # truncate_before_load: bool = False, this must be forced as False with delta exp?
```

Comment:
This should not be a commented out parameter, it should just be removed.  In testing it doesn't seem like passing this as True or False in the config has any effect (truncates never happen).